src/js/components/streaming-api-handler.tsx

```diff
@@ -372,10 +386,6 @@ class StreamingAPIHandler extends React.Component<Props, void> {
   }
 }
 
-const mapStateToProps = (state: StateTree): GeneratedStateProps => ({
-  team: User.selectors.getTeam(state),
-});
-
 const mapDispatchToProps = (dispatch: Dispatch<any>): GeneratedDispatchProps => ({
```

Comment:
Not easily—we don't have a type that covers all actions. At some point we should improve typings for actions in general in order to get type-safe reducers.